translations/ui-lists/en.json

```diff
@@ -170,5 +170,19 @@
   "missing-all-entity-type-permissions.heading": "You do not have the required permissions to use the Lists app",
   "missing-all-entity-type-permissions.message": "Please contact your system administrator for help",
 
-  "app-menu.keyboard-shortcuts": "Keyboard shortcuts"
+  "app-menu.keyboard-shortcuts": "Keyboard shortcuts",
+
+  "commands-label.create": "Create a new list",
+  "commands-label.edit": "Edit a list",
+  "commands-label.save": "Save a list" ,
+  "commands-label.duplicate": "Duplicate a list",
+  "commands-label.toggle-lists-detail-accordion": "Expand or collapse a Lists detail accordion",
+  "commands-label.expand-list-detail-accordions": "Expand all List detail accordions",
+  "commands-label.collapse-list-detail-accordions": "Collapse all List detail accordions",
+  "commands-label.go-to-filter-pane": "Go to Lists Filter pane",
+  "commands-label.close-modal-pop-up": "Close a modal or pop-up",
```

Comment:
Modal probably isn't a term that is familiar to most non-devs. Is there another word we could use?

Reply:
I looked in other folio apps, and it looks like they word it the same, so I'd leave it as is :)